src/lib/stitching/mergeSchemas.ts

```diff
@@ -56,7 +56,7 @@ export const incrementalMergeSchemas = (localSchema, version: 1 | 2) => {
   schemas.push(diffusionSchema)
 
   useStitchingEnvironment(
-    causalityStitchingEnvironment({ causalitySchema, localSchema })
+    causalityStitchingEnvironment({ causalitySchema, localSchema, version })
```

Comment:
```suggestion
    causalityStitchingEnvironment({ causalitySchema, localSchema })
```

Same as mentioned above

Reply:
there was one other place that needed it so i applied these undos locally and pushed back.